pkg/operator/ceph/object/status.go

```diff
@@ -49,6 +49,9 @@ func updateStatusPhase(client client.Client, namespacedName types.NamespacedName
 	}
 
 	objectStore.Status.Phase = status
+	if len(info) != 0 {
```

Comment:
Why not just always assign it? If the length is 0 is won't make a difference.

Reply:
Good catch, not sure why I did that :)